mail/1.6/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Jakarta Mail Specification 1.6"
+date: 2019-07-12T14:29:05-0700
+summary: "First release for Jakarta EE 8"
+---
+Jakarta Servlet defines server-side handling for HTTP requests and responses.
```

Comment:
I think this should be related to JavaMail not Servlet?

Reply:
Oops, fixed!